.github/workflows/pr-merge-conflict.yml

```
@@ -0,0 +1,92 @@
+# This action is centrally managed in https://github.com/asyncapi/.github/
+# Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in above mentioned repo
+
+# This action handles merge conflicts and provides auto-rebase functionality for pull requests
+name: Merge Conflict and Auto-Rebase 
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, edited, ready_for_review]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-merge-conflicts:
+    if: github.event_name == 'pull_request_target'
+    runs-on: ubuntu-latest
+    outputs:
+      conflicts: ${{ steps.conflicts.outputs.conflicts }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check for merge conflicts
+        id: conflicts
+        run: |
+          git fetch origin ${{ github.base_ref }}
+          git merge origin/${{ github.base_ref }} --no-commit --no-ff
+          if [[ $(git ls-files -u | wc -l) -gt 0 ]]; then
+            echo "Merge conflicts detected."
+            echo "::set-output name=conflicts::true"
+            git merge --abort
+          else
+            echo "No merge conflicts."
+            echo "::set-output name=conflicts::false"
+          fi
+      - name: Post Merge Conflict Notification
+        if: steps.conflicts.outputs.conflicts == 'true'
+        uses: marocchino/sticky-pull-request-comment@v2.5.0
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

this way our bot is author of comment